src/More/About.tsx

```diff
@@ -23,12 +25,17 @@ export const AboutScreen: FunctionComponent = () => {
     t("about.description", { applicationName, healthAuthorityName }),
   )
 
+  const authorityLinks = useAuthorityLinks("about", localeCode)
```

Comment:
What do you think about a flow more like:

```typescript
import links from "./links.json"

...

const aboutLinks = links.about
const displayableLinks = applyTranslations(aboutLinks, localeCode)
```

Having the importing of the data in the same spot as the apply the translation seems a bit overly abstracted to me.

Reply:
the link data could live in the config context as well. which might be a better place to map the data to apply the label translation.